java/client/src/main/java/glide/api/models/commands/function/FunctionListOptions.java

```diff
@@ -0,0 +1,15 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.function;
+
+/**
+ * Option for <code>FUNCTION LIST</code> command.
```

Comment:
should it be {@link} for the command name here?

Reply:
Added